pkgs/development/python-modules/sphinxcontrib-towncrier/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, towncrier
+, tomli
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-towncrier";
+  version = "0.3.2a0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-fPGKycE1M//9t5TasnjGrYRfjIW/pWlC66B31UOIX6Y=";
+  };
+
+  propagatedBuildInputs = [ sphinx towncrier tomli ];
+
+  pythonImportsCheck = [ "sphinxcontrib.towncrier" ];
+
+  meta = with lib; {
+    description = "Towncrier extension for Sphinx";
+    homepage = "https://github.com/sphinx-contrib/sphinxcontrib-towncrier";
+    maintainers = with maintainers; [ ];
```

Comment:
No maintainers?

Reply:
Dropping. Looking around it's more maintained than unmaintained sphinxcontrib modules. I won't make it worse.